<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvaluatePage</class>
 <widget class="QWidget" name="EvaluatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>1500</width>
     <height>920</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{border-top-left-radius: 8px;border-top-right-radius:8px;background: #FFFFFF;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="display_Label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>654</width>
      <height>630</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#E4F4FC;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name1_Label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>200</y>
      <width>230</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下限值</string>
    </property>
   </widget>
   <widget class="QLabel" name="name2_Label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>330</y>
      <width>230</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上限值</string>
    </property>
   </widget>
   <widget class="QLabel" name="minValue_Label">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>200</y>
      <width>241</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>minValue</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxValue_Label">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>340</y>
      <width>241</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>maxValue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startEvaluate_Btn">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>790</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 140px;
height: 70px;
background: #0D9DDB;
border-radius: 8px;
width: 64px;
height: 32px;
font-size: 32px;
font-family: SourceHanSansCN-Medium, SourceHanSansCN;
font-weight: 500;
color: #FFFFFF;
line-height: 48px;</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{background: #0D9DDB;
border-top-left-radius: 8px;border-top-right-radius:8px}</string>
    </property>
    <widget class="QPushButton" name="close_Btn">
     <property name="geometry">
      <rect>
       <x>1430</x>
       <y>20</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/close.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="title_Label">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>12</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>评估</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
